ocs_ci/deployment/deployment.py

```diff
@@ -490,6 +573,11 @@ def deploy_ocs(self):
             return
         except (IndexError, CommandFailed):
             logger.info("Running OCS basic installation")
+
+        if config.DEPLOYMENT['independent_mode']:
+            logger.info("Deploying OCS on independent mode RHCS")
+            return self.deploy_with_independent_mode()
```

Comment:
Before deploying OCS on independent cluster, can we check whether external ceph cluster in good shape ( may be ceph health )

Reply:
I can't think of any straight forward solutions to do this unless we have either ssh with public/private key and then connecting to remote machine. Any idea ?

Reply:
I though we have public to connect to external cluster

Reply:
Same.
For Health: Will we fail the deployment if it is not HEALTH_OK ?
reason: It may happen that the RHCS cluster is in Warn state due to object counts, Slow IO on MDS etc.. Will we fail the deployment even for these issues ?
We can discuss on this if need be,

Reply:
@nehaberry sure, we can have discussion on this.

Reply:
@vavuthu , I can add this check when we have RHCS deployment integrated. As of now user has to deploy RHCS externally and then we will run OCS-CI for OCS deployments. WDYT ?

Reply:
> @vavuthu , I can add this check when we have RHCS deployment integrated. As of now user has to deploy RHCS externally and then we will run OCS-CI for OCS deployments. WDYT ?

we can take up this later